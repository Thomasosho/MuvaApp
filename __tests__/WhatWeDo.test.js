jest.useFakeTimers("modern");
import React from "react";
import renderer from "react-test-renderer";
import WhatWeDoScreen from "../src/screens/what-we-do/WhatWeDo";

it("renders correctly", () => {
  const tree = renderer.create(<WhatWeDoScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
