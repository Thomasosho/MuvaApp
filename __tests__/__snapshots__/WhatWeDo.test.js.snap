// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 16,
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#000",
        "fontSize": 14,
        "textAlign": "justify",
      }
    }
  >
    We buckle down to remain the best mobile apps development company in Kenya and beyond. You said you like details? Click any link below:
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "center",
        "backgroundColor": "#000",
        "borderRadius": 20,
        "justifyContent": "center",
        "opacity": 1,
        "padding": 15,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#fff",
          "fontSize": 16,
        }
      }
    >
      Go Back
    </Text>
  </View>
</View>
`;
